pkg/gotemplate/new.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	minGoVersion  = "1.15"
+	minGoVersion  = "1.20"
```

Comment:
```suggestion
	minGoVersion  = "1.21"
```

This should probably also be 1.21, as we want to implement slog (#311), which is only available in Go >= 1.21

Reply:
Thank you :) done in here as well